datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -113,6 +115,11 @@ public SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength Str
         String userCrn = ThreadBasedUserCrnProvider.getUserCrn();
         Pair<SdxCluster, FlowIdentifier> result = sdxService.createSdx(userCrn, name, createSdxClusterRequest, null);
         SdxCluster sdxCluster = result.getLeft();
+
+        SdxDatabaseRequest databaseRequest = new SdxDatabaseRequest();
+        databaseRequest.setAvailabilityType(SdxDatabaseAvailabilityType.NONE);
+        createSdxClusterRequest.setExternalDatabase(databaseRequest);
+
```

Comment:
is this intentional? I think we still need database for datalake by default

Reply:
This is not. This was probably added for internal testing. 
Bryce, Please remove the change.